lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -183,6 +183,10 @@ def text_2d_to_3d(obj, z=0, zdir='z'):
 class Line3D(lines.Line2D):
     """
     3D line object.
+
+    .. note:: Use `get_data_3d` to obtain the data associated with the line.
+            `~.Line2D.get_xdata` and `~.Line2D.get_ydata` will return the x- and
+            y-coodinates of the projected line, not the x- and y-data of the 3D-line.
```

Comment:
Should this mention the setters as well, for completeness?

Reply:
Good idea! I'll add it.

Reply:
```suggestion
            y-coordinates of the projected line, not the x- and y-data of the 3D-line.
```

Reply:
Whoops! Thanks!